### ISSUE TITLE
TCP port 23 (telnet) being used?

### ISSUE THREAD

**Issue by gngrossi** [ZOSOpenTools/c3270port#4]
Connection failed:
xxxxxxxx, port 23: EDC8128I Connection refused.
Disconnected.

**Comment by IgorTodorovskiIBM** [comment]
I wonder if this comment from the [blog](https://igortodorovskiibm.github.io/blog/2023/06/25/c3270-on-zos/) helps:
> One small remark - most if not all z/OS installations have (should have) closed down their open ports on TN3270 (the server which serves 3270 connections as c3270 uses). To TLS connect to the secured port you need to add -port <secureport> -cafile <ca.pem> to connect securely to TN3270 using c3270.

**Comment by gngrossi** [comment]
Thanks for the information...closing the issue.

I'm running into an issue that pertains to a self-signed certificate error which is our issue. I have examined the trace file showing the same stdout messages.

Also, I used openssl to connect to the server and it does connect but shows the same error.

bash-5.2$ openssl s_client -connect <host>:2323
CONNECTED(00000004)
verify error:num=19:self signed certificate in certificate chain

--

bash-5.2$ c3270 <host>:2323 -trace -tracefile /SYSA/tmp/gary/c3270.trc2 -cadir /hewitt/apps/scripts/JavaKeyStore
c3270 v4.2ga10 Fri Jun 23 20:53:33 UTC 2023 JENKINS
Copyright 1989-2023 by Paul Mattes, GTRC and others.

Type 'show copyright' for full copyright information.
Type 'help' for help information.
Connection failed:
TLS: Host certificate verification failed:
self signed certificate in certificate chain (19)
Disconnected.

--

For now, I'm using -noverifycert to continue testing.
